### DISCUSSION TITLE
Breadcrumbs for Laravel routes

### DISCUSSION THREAD

**Discussion by Christophvh** [statamic/cms#4891] [Q&A]
I know about the breadcrumbs helper for Statamic related pages.

But our website uses a combination of both Laravel pages (with controllers) and Statamic pages.

Is there a possibility to extend or customize this behavior (maybe in a controller?) so breadcrumbs work for laravel routes too?

The CP already has this: https://statamic.dev/extending/breadcrumbs

**Comment by edalzell** [comment]
you'll have to make your own tag to handle those other routes.
